server/konnectors/withings.coffee

```diff
@@ -270,3 +303,52 @@ saveMeasures = (measures, callback) ->
             BloodPressure.all (err, bloodPressures) ->
                 return callback err if err
                 processData scaleMeasures, heartBeats, bloodPressures
+
+
+saveActivityMeasures = (measures, callback) ->
+
+    console.log 'Processing activity measures...'
+    processData = (stepsMeasures) ->
+
+        stepsHash = hashMeasuresByDate stepsMeasures
+
+        newSteps = measures.body?.series?.type_36
+        newDistances = measures.body?.series?.type_40
+
+        return callback() if not newSteps? and not newDistances?
+
+        stepsToSave = []
+        for date, valueObj of newSteps
+            dateAsMom = moment date
+            steps = valueObj.sum
+            if not stepsHash[dateAsMom]?
+                stepMeasure = new Steps
+                stepMeasure.date = dateAsMom
+                stepMeasure.steps = steps
+                stepMeasure.vendor = 'Withings'
+                if newDistances[date]?
+                    stepMeasure.distance = newDistances[date].sum
+                stepsToSave.push stepMeasure
+
+        console.log "Found #{stepsToSave.length} new steps measures to save!"
+        saveInstance = (model, cb) ->
+            model.save cb
+        async.forEach stepsToSave, saveInstance, (err) ->
+            return callback err if err?
+
+            console.log 'Steps measures saved.'
+            notifContent = null
+            if stepsToSave.length
+                localizationKey = 'notification withings'
+                options = smart_count: stepsToSave.length
+                notifContent = localization.t localizationKey, options
+            callback null, notifContent
+
+        callback()
+        return
+
+    console.log 'Fetching former activity measures...'
+    Steps.all (err, stepsMeasures) ->
+        return callback err if err
+        console.log 'Steps', stepsMeasures
```

Comment:
Removed this one and the same in the body measures (there was one such log for the scales)